src/hw9/double.c

```diff
@@ -0,0 +1,42 @@
+#include "double.h"
+#include <stdint.h>
+#include <stdio.h>
+
+typedef union {
+  double value;
+  uint64_t bits;
+} DoubleUnion;
+
+void analyzeDouble(double number) {
+  DoubleUnion du;
+  du.value = number;
+  uint64_t bits = du.bits;
+
+  int sign = (bits >> 63) & 1;
```

Comment:
Лучше так:
```suggestion
  bool isNegative = (bits >> 63) == 1;
```
если потом используется в тернарном операторе